metadata-ingestion/src/datahub/ingestion/source/sql/oracle.py

```diff
@@ -608,6 +619,16 @@ def get_db_name(self, inspector: Inspector) -> str:
         return db_name
 
     def get_inspectors(self) -> Iterable[Inspector]:
+        # if connecting to oracle with thick_mode client it must be initialized before calling
+        # create_engine, which is called in super().get_inspectors()
+        if self.config.thick_mode:
```

Comment:
not sure when `get_inspectors` gets called, so wondering if there is a better place for the setup of the oracle client library. maybe in the initialization of the `OracleSource`?